### PR TITLE
fix: translate a section title on Why Notify

### PR DESCRIPTION
**Pull request by AntoineAugusti**
The translation string had an extra quote, therefore it's not translated at the moment. See https://notification.canada.ca/why-notify in production

Screen Shot 2021-03-16 at 20 51 06

### CODE REVIEW

File: app/translations/csv/fr.csv [hunk 1]
```diff
@@ -1106,7 +1106,7 @@
...
 "Conditional text","Texte conditionnel"
-"To add optional conditional text, in double brackets, use two question marks after your condition variable and before the text that appears if the condition is met. For example:",Pour ajouter du texte conditionnel facultatif, utilisez deux points d'interrogation après votre condition et avant le texte qui devra apparaître si la condition est remplie, et mettez le tout entre parenthèses doubles. Par exemple :"
+"To add optional conditional text, in double brackets, use two question marks after your condition variable and before the text that appears if the condition is met. For example:","Pour ajouter du texte conditionnel facultatif, utilisez deux points d'interrogation après votre condition et avant le texte qui devra apparaître si la condition est remplie, et mettez le tout entre parenthèses doubles. Par exemple :"
```
Comment (AntoineAugusti) [inline app/translations/csv/fr.csv:1109]: Missing double quote here, unrelated

**Review by anikbrazeau — APPROVED** [review]
MERCI!!! 🚀

File: app/translations/csv/fr.csv (hunk 1 above)
Comment (anikbrazeau) [reply app/translations/csv/fr.csv:1109]: I'll try to do an audit/review of our translation file in the next week or two 🙈

**Comment by anikbrazeau** [comment]
[Content updates to clarify trail mode](https://trello.com/c/N5fK1VMH/390-content-updates-to-clarify-trail-mode)

**Pull request merged**